src/test/java/cmput402/tdd/AppTest.java

```diff
@@ -210,42 +210,42 @@ public void testEditShoppingList(){
         assertEquals(1, person.getShoppingLists().get("list1").getItems().size());
     }
 
-    @Test
-    public void testCreateRecipe() {
-        App app = new App();
-        Recipe recipe;
-        
-        //Test1: correct input
-        ByteArrayInputStream in = new ByteArrayInputStream("recipe1\n".getBytes());
-        Scanner scanner = new Scanner(in); 
-        try{
-            recipe = app.createRecipe(scanner);
-            Assert.assertNotNull(recipe);
-            assertEquals(recipe.getName(), "recipe1");
-        } catch (Exception e) {
-            Assert.fail();
-        }
-
-        //Test2: empty name given
-        in = new ByteArrayInputStream("\n".getBytes());
-        scanner = new Scanner(in); 
-        try{
-            recipe = app.createRecipe(scanner);
-            fail();
-        } catch (Exception e){
-            assertEquals("Name cannot be empty.", e.getMessage());
-        }
-
-        //Test3: list name > 40 characters given
-        in = new ByteArrayInputStream("12121212121212121212121212121212121212121\n2\n".getBytes());
-        scanner = new Scanner(in); 
-        try{
-            recipe = app.createRecipe(scanner);
-            Assert.fail();
-        } catch (Exception e){
-            assertEquals("Recipe name cannot be longer than 40 characters.", e.getMessage());
-        }
-    }
+//    @Test
```

Comment:
Why is this test completely commented out?

Reply:
My mistake, it was because CreateRecipe function is not in my branch which caused my build to fail

Reply:
He is correct while we haven't utilized it as such it should be public for future use.